core/jreleaser-model-impl/src/main/java/org/jreleaser/model/internal/validation/announce/MastodonAnnouncerValidator.java

```diff
@@ -50,10 +53,15 @@ public static void validateMastodon(JReleaserContext context, MastodonAnnouncer
                 errors,
                 context.isDryrun()));
 
-        if (isBlank(mastodon.getStatus())) {
+        if (isBlank(mastodon.getStatus()) && isBlank(mastodon.getStatusTemplate()) && mastodon.getStatuses().isEmpty()) {
             mastodon.setStatus(RB.$("default.release.message"));
         }
 
+        if (isNotBlank(mastodon.getStatusTemplate()) &&
```

Comment:
The validation for multiple statuses must happen before the single status. See `org.jreleaser.model.internal.validation.announce.TwitterAnnouncerValidator`